src/components/Trade/hooks/useSwapper/useSwapper.tsx

```diff
@@ -137,15 +137,14 @@ export const useSwapper = () => {
   ])
 
   useEffect(() => {
-    ;(async () => {
-      flags && setSwapperManager(await getSwapperManager(flags))
-    })()
+    if (!flags) return
+
+    getSwapperManager(flags).then(setSwapperManager)
```

Comment:
tyvm, we tend to avoid `.then().catch()` in profit of `await` and try/catches but in this case, absolutely yes